R/ebp.R

```diff
@@ -84,15 +84,24 @@
 #' \code{\link[parallelMap]{parallelStart}}.
 #' @param custom_indicator a list of functions containing the indicators to be
 #' calculated additionally. Such functions must and must only depend on the
-#' target variable \code{y} and the \code{threshold}.
+#' target variable \code{y}, \code{weights} and the \code{threshold}.
 #' Defaults to \code{NULL}.
 #' @param na.rm if \code{TRUE}, observations with \code{NA} values are deleted
 #' from the population and sample data. For the EBP procedure complete
 #' observations are required. Defaults to \code{FALSE}.
+#' @param aggregate_to a character string containing the name of a variable from
+#' population data that indicates the target area level for which the
+#' results are to be displayed. The variable can be numeric or a factor.
+#' Defaults to \code{NULL}.
 #' @param weights a character string containing the name of a variable that
 #' indicates weights in the sample data. If a character string is provided
-#' a weighted version of the ebp will be used.The variable has to be numeric.
+#' a weighted version of the ebp will be used. The variable has to be numeric.
 #' Defaults to \code{NULL}.
+#' @param pop_weights a character string containing the name of a variable that
```

Comment:
Is it clear what population weights are? And what does it mean that it needs to be provided for each bootstrap?

Reply:
Ich habe das leicht verändert und in der Erweiterungs-Vignette wird das mit Beispielen ganz genau erklärt. 
**Dropbox\emdi paper\extensions**